vite.config.ts

```diff
@@ -38,6 +37,7 @@ export default defineConfig({
             "@hooks": resolve(__dirname, "lib/hooks"),
             "@providers": resolve(__dirname, "lib/providers"),
             "@utils": resolve(__dirname, "lib/utils"),
+            "@styles": resolve(__dirname, "lib/styles"),
```

Comment:
shall we remove the `@quill-bundle` in line 33 as it's no longer exist?

Reply:
yes @Nuzhy-Deriv  that's true, good catch!